.changeset/angry-jeans-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Add support for permissions to the DefaultCatalogCollator by adding the visibilityPermission prop and including authorization.resourceRef on indexed documents.
```

Comment:
```suggestion
Add support for permissions to the DefaultCatalogCollator.
```

^ On this and the TechDocs backend: does it make sense to leave off the implementation details in the changelog?  Thinking the audience for these is app integrators who use (rather than developers who build) on these things.